pig-tests/src/org/partiql/pig/tests/generated/sample-universe.kt

```diff
@@ -7318,10 +7318,12 @@ class DomainA private constructor() {
          */
```

Comment:
changes to this file are not relevant in this PR since the templates didn't change.  it's easier to view the changes I made to the `sample-universe.ion` file instead.